google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddRemarketingAction.java

```diff
@@ -0,0 +1,152 @@
+        RemarketingAction.newBuilder()
+            .setName(StringValue.of("Remarketing action #" + System.currentTimeMillis()))
+            .build();
+
+    // Creates a remarketing action operation.
+    RemarketingActionOperation operation =
+        RemarketingActionOperation.newBuilder().setCreate(remarketingAction).build();
+
+    // Issues a mutate request to add the remarketing action and prints out some information.
+    String remarketingActionResourceName;
+    try (RemarketingActionServiceClient conversionActionServiceClient =
+        googleAdsClient.getLatestVersion().createRemarketingActionServiceClient()) {
+      MutateRemarketingActionsResponse response =
+          conversionActionServiceClient.mutateRemarketingActions(
+              Long.toString(customerId), Collections.singletonList(operation));
+      remarketingActionResourceName = response.getResults(0).getResourceName();
+      System.out.printf(
+          "Added remarketing action with resource name '%s'.%n", remarketingActionResourceName);
+    }
+
+    // Creates a query that retrieves the previously created remarketing action with its generated
+    // tag snippets.
+    String query =
+        String.format(
+            "SELECT remarketing_action.id, remarketing_action.name,"
+                + " remarketing_action.tag_snippets "
+                + "FROM remarketing_action "
+                + "WHERE remarketing_action.resource_name = '%s'",
+            remarketingActionResourceName);
+    try (GoogleAdsServiceClient googleAdsServiceClient =
+        googleAdsClient.getLatestVersion().createGoogleAdsServiceClient()) {
+      // Issues a search request.
+      SearchPagedResponse searchPagedResponse =
+          googleAdsServiceClient.search(Long.toString(customerId), query);
+
+      // There is only one row because we limited the search using the resource name, which is
+      // unique.
+      GoogleAdsRow googleAdsRow = searchPagedResponse.iterateAll().iterator().next();
+
+      // Prints some attributes of the remarketing action. The ID and tag snippets are generated by
+      // the API.
+      RemarketingAction newRemarketingAction = googleAdsRow.getRemarketingAction();
+      System.out.printf(
+          "Remarketing action has ID %d and name '%s'.%n%n",
+          newRemarketingAction.getId().getValue(), newRemarketingAction.getName().getValue());
+      System.out.println("It has the following generated tag snippets:");
+      for (TagSnippet tagSnippet : newRemarketingAction.getTagSnippetsList()) {
+        System.out.printf(
+            "Tag snippet with code type '%s' and code page format '%s' has the following global"
+                + " site tag:%n%s%n",
+            tagSnippet.getType(),
+            tagSnippet.getPageFormat(),
+            tagSnippet.getGlobalSiteTag().getValue());
+        System.out.printf(
+            "and the following event snippet:%n%s%n", tagSnippet.getEventSnippet().getValue());
```

Comment:
Missing one "%n" at the end based on the [golden](https://github.com/googleads/google-ads-php/blob/master/examples/Remarketing/AddRemarketingAction.php#L172)